src/Pim/Bundle/UserBundle/Doctrine/ORM/Repository/GroupRepository.php

```
@@ -24,6 +34,16 @@ public function findOneByIdentifier($code)
         return $this->findOneBy(['name' => $code]);
     }
 
+    /**
+     * Get the default user group
+     *
+     * @return null|object
+     */
+    public function getDefaultUserGroup()
+    {
+        return $this->findOneByIdentifier(User::GROUP_DEFAULT);
```

Maybe this function should throw an exception instead of returning null. Having no default group will explode the rest of the application.

---

yes, you are right but it is not the scope of this PR. Here, I wanted to clean my first PR. I did not introduce this method.